client/payment-request/index.js

```diff
@@ -436,11 +438,23 @@ jQuery( ( $ ) => {
 			} );
 		},
 
+		getElements: () => {
+			return $(
+				'#wcpay-payment-request-wrapper,#wcpay-payment-request-button-separator'
```

Comment:
Should we store this as constants describing the element? Would be more readable then.

Reply:
That was the idea behind introducing `getElements` as a new method, which is the only place those IDs are included. Based on most jQuery code I've seen, this is a fairly normal practice, rather than declaring the selector as a constant and then querying it. i can go either way here, but this change seems unnecessary.